app/models/estimate_model.rb

```diff
@@ -39,4 +41,9 @@ def partner_owns_property?
   def controlled?
     level_of_help == "controlled"
   end
+
+  def upper_tribunal?
+    # controlled_proceeding_type != "SE003"
+    controlled_proceeding_type.in?(%w[IM030 IA031])
```

Comment:
Let's make this method safer - at the moment this will explode if controlled_proceeding_type is nil. Maybe `controlled_proceeding_type&.in?(...)`?